### ISSUE TITLE
Suggested URI in ConsoleApplication readme.md file is missing on Azure Redirect URIs page.

### ISSUE THREAD

**Issue by aussiesim** [microsoftgraph/msgraph-training-building-apps#13]
I followed the tutorial in the readme file:

https://github.com/microsoftgraph/msgraph-training-building-apps/blob/master/Demos/01-console-application/README.md

In the "Register the Application" section, step 4 implies that **urn:ietf:wg:oauth:2.0:oob** should be listed as a suggested URI but it wasn't available. I added it as a "Public client/native (mobile & desktop)" URI instead and all is working fine.

I would suggest rewording the readme file to reflect this.

Thanks!

**Comment by jasonjoh** [comment]
Closing this issue as we are archiving this repository and will not be maintaining it going forward. For an updated sample that covers these topics, please see:

- Console app: https://github.com/microsoftgraph/msgraph-training-dotnet-core
- JavaScript app: https://github.com/microsoftgraph/msgraph-training-javascriptspa
- Azure Function: https://github.com/microsoftgraph/msgraph-training-azurefunction-csharp
- Xamarin app: https://github.com/microsoftgraph/msgraph-training-xamarin

We apologize for any inconvenience.
